examples/jsm/webxr/OculusHandModel.js

```diff
@@ -0,0 +1,203 @@
+import { Object3D, Sphere, Box3 } from "../../../build/three.module.js";
+import { FBXLoader } from "../loaders/FBXLoader.js";
+
+const JOINTS = [
+  "wrist",
+  "thumb-metacarpal",
+  "thumb-phalanx-proximal",
+  "thumb-phalanx-distal",
+  "thumb-tip",
+  "index-finger-metacarpal",
+  "index-finger-phalanx-proximal",
+  "index-finger-phalanx-intermediate",
+  "index-finger-phalanx-distal",
+  "index-finger-tip",
+  "middle-finger-metacarpal",
+  "middle-finger-phalanx-proximal",
+  "middle-finger-phalanx-intermediate",
+  "middle-finger-phalanx-distal",
+  "middle-finger-tip",
+  "ring-finger-metacarpal",
+  "ring-finger-phalanx-proximal",
+  "ring-finger-phalanx-intermediate",
+  "ring-finger-phalanx-distal",
+  "ring-finger-tip",
+  "pinky-finger-metacarpal",
+  "pinky-finger-phalanx-proximal",
+  "pinky-finger-phalanx-intermediate",
+  "pinky-finger-phalanx-distal",
+  "pinky-finger-tip",
+];
+const TOUCH_RADIUS = 0.01;
+const POINTING_JOINT = "index-finger-tip";
+
+class XRHandOculusMeshModel {
+  constructor(handModel, controller, handedness) {
+    this.controller = controller;
+    this.handModel = handModel;
+
+    this.bones = [];
+    const loader = new FBXLoader();
+
+    // loader.setPath("");
+    loader.load("models/fbx/OculusHandNew_" + handedness + ".fbx", (object) => {
```

Comment:
Is it possible to use `GLTFLoader` here, too? It seems the FBX assets can then be removed.